packages/core/content-releases/admin/src/index.ts

```diff
@@ -1,13 +1,36 @@
 import { prefixPluginTranslations } from '@strapi/helper-plugin';
+import { PaperPlane } from '@strapi/icons';
+
+import { pluginId } from './pluginId';
+
+import type { Plugin } from '@strapi/types';
 
 // eslint-disable-next-line import/no-default-export
-export default {
-  register() {
+const admin: Plugin.Config.AdminInput = {
```

Comment:
Why do you need to define the response type? 🤔

Reply:
I followed what was done in the email plugin which is one of the few plugins converted in TS https://github.com/strapi/strapi/blob/9e14912c58fadb39fda6d8e78015c27eed9a960c/packages/core/email/admin/src/index.ts#L14